_maps/map_files/RandomRuins/SpaceRuins/deepstorage.dmm

```diff
@@ -342,14 +337,14 @@
 "aH" = (
 /obj/machinery/vending/clothing,
 /turf/simulated/floor/wood{
-	tag = "icon-wood-broken";
-	icon_state = "wood-broken"
+	icon_state = "wood-broken";
+	tag = "icon-wood-broken"
```

Comment:
You got any way of cleansing this `tag` nonsense as well?

Reply:
hmm, are they even used in anything useful?

Reply:
ah, they are used for landmarks to specify spawn points, so filtering them out is a bit difficult

Reply:
Having said that, all of those are `tag = "start*[taghere]"` so you could run a reasonably easy regex to keep these and delete the others